templates/problem/problem.html

```diff
@@ -238,6 +238,30 @@ <h2 style="color:#393630; display: inline-block">{{ title }}</h2>
             </div>
         </div>
     {% endif %}
+    {% if problem.is_editable_by(request.user) %}
+        <div id="available-judges">
+            <i class="fa fa-database fa-fw"></i><span
+                class="pi-name">{% trans trimmed count=judges|length %}
+            Judge:
+            {% pluralize count %}
+            Judges:
+        {% endtrans %}</span>
+            <div class="pi-value judges-value">
+                {% if judges %}
+                    {% if perms.judge.change_judge %}
+                        {% for judge in judges %}
+                            <a href="{{ url('admin:judge_judge_change', judge.id) }}">{{ judge.name }}</a>
+                            {% if not loop.last %}, {% endif %}
+                        {% endfor %}
+                    {% else %}
+                        {{ judges|join(", ") }}
+                    {% endif %}
+                {% else %}
+                    <i>{{ _('No available judges') }}</i>
```

Comment:
Probably "none available" is fine here, since we already have `Judge:` preceding.